prow/github/client.go

```diff
@@ -651,3 +652,41 @@ func (c *Client) FindIssues(query string) ([]Issue, error) {
 	}, &issSearchResult)
 	return issSearchResult.Issues, err
 }
+
+// set the trusted orgs
+func (c *Client) SetTrustedOrgs(trustedOrgsList string) {
+	c.trustedOrgs = strings.Split(trustedOrgsList, ",")
+}
+
+// return true if an org is in the trusted list
+func (c *Client) IsTrustedOrg(org string) bool {
+
+	if c.trustedOrgs == nil {
```

Comment:
ok - will remove.  NullPointerException is burned into my consciousness from Java.